Dockerfile

```diff
@@ -1,7 +1,10 @@
-FROM ubuntu:latest
+FROM --platform=linux/amd64 ubuntu:latest
```

Comment:
perhaps only valid for Mac M1, and would not apply to others

https://stackoverflow.com/questions/71040681/qemu-x86-64-could-not-open-lib64-ld-linux-x86-64-so-2-no-such-file-or-direc